### DISCUSSION TITLE
Desafio 30 Dias de Vanilla JS

### DISCUSSION THREAD

**Discussion by baltazarparra** [frontendbr/forum#388] [Dicas]
Repassando uma referência muito foda do Wes Bos, pra galera que está estudando javascript.
[https://javascript30.com/](https://javascript30.com/)

Construindo 30 coisas em 30 dias com 30 tutoriais
Sem framework, sem compiler, sem lib e sem boilerplate.
[https://javascript30.com/](https://javascript30.com/)

**Comment by victormiguez** [comment]
Que ideia foda, valeu por compartilhar!

**Comment by andrewsfg** [comment]
Genial, muito obrigado.

**Comment by brunoalmeida-dev** [comment]
Eu tava esperando essa série dele 💃 

É muito bom também a série de Flexbox que ele possui, se não me engano é free ainda.

**Comment by andrewsfg** [comment]
@brunob182 Você tem o link fácil aí? Obrigado

**Comment by brunoalmeida-dev** [comment]
@andrewsfg da uma olhada http://flexbox.io/

**Comment by gabrieldarezzo** [comment]
`Please don't make me do another todo app.` HSAUHSUAHASUAS

Ideia apoiada xD (To de ferias da facu mesmo :stuck_out_tongue: )

Edit:

Repo do Meu xD
https://github.com/gabrieldarezzo/JavaScript30

**Comment by diogomoretti** [comment]
Eu vi e assisti assim que saiu também... mas não gostei pelo simples fato de simplesmente ele sair colocando código global de qualquer jeito... Seria bacana ele aplicar _patterns_ e _good practices_.

**Comment by gabrieldarezzo** [comment]
@diogomoretti que tal participar? 
Da um exemplo com  _patterns_ e good _practices_.

**Comment by diogomoretti** [comment]
@gabrieldarezzo eu tava falando do "curso" em si.... que o WB poderia ter feito um pouco diferente :stuck_out_tongue_winking_eye:

**Comment by gabrieldarezzo** [comment]
@diogomoretti Entendi...
Mas participa mesmo assim pow haha
To doido pra ver algum monstro do @braziljs fazendo o desafio.

**Comment by diogomoretti** [comment]
@gabrieldarezzo haha, que nada cara... monstro é o @sindresorhus & CIA. Participar que você diz é abrir um repo para colocar os 30 dias lá?

**Comment by gabrieldarezzo** [comment]
@diogomoretti Isso ai alguma ideia em Js-Vanilla que da pra fazer em um dia? :)

**Comment by diogomoretti** [comment]
@gabrieldarezzo você pode pegar esse [site de plugins js](http://unheap.com) (onde 90% é jQuery) e transformá-los para vanilla, penso que é um bom aprendizado... além do mais, você estará fazendo coisas práticas do dia-dia :stuck_out_tongue_winking_eye:

**Comment by felipefialho** [comment]
@diogomoretti Mitou. 

Sem contar que ainda contribui com uma penca de plugins open source vanilla 🥇

**Comment by baltazarparra** [comment]
@diogomoretti Ele deixa claro isso desde o começo.

"We steer clear of opinionated design patterns, frameworks, libraries and any type of abstraction. These things are great and often needed, but can obstruct learning and constrain initial understanding."

E posto desta forma, faz total sentido, pelo menos pra mim.

**Comment by diogomoretti** [comment]
@baltazarparra eu sei que ele _setou_ as expectativas. Assisti os vídeos sabendo disso. O que comentei é que não acho tão rica esse tipo de abordagem. Vale a pena assistir? Vale sim, só que poderia ser mais enriquecedor =]
